app/models/button_contact.rb

```diff
@@ -0,0 +1,13 @@
+class ButtonContact
+  include ButtonMaker
+
+  def html_button
+    return unless eligible?
+    "<a class='btn button-secondary button-small' " \
+      "href='https://libraries.mit.edu/archives/'>Contact Us</a>"
+  end
+
+  def eligible?
```

Comment:
ahhhhhh yeah I like so much how this refactor lets us have a simple `eligible?` interface.

Reply:
Except on the HoldRecall :/ I tried to make that the same interface but it was too big of a refactor to keep the tests reasonable static for these changes.

Reply:
Yeah :/ I hear you and I think it's cool once documented.